samples/code-samples/ChangeFeedProcessor/Readme.md

```diff
@@ -0,0 +1,2 @@
+If you are using Change FeedProcessor library earlier than 2 then please take a look at the code commit on 
```

Comment:
It's not clear how this project is related to CFP library. Is it using CFP or provides functionality similar to CFP? I would recommend in readme explain in short what this project is for, what is the goal that we have this project.
Also what if I use v2 version, where would I go? Also I think v2 is more readable than version 2.